charts/otv-backend/values.yaml

```
@@ -21,7 +21,13 @@ config: |
         "wsEndpoint": "wss://kusama-rpc.polkadot.io/"
     },
     "db": {
-        "storageFile": "/data/validator.db"
+        "migrate": true,
```

```suggestion
        "migrate": false,
```

We should set this now to false, since the migration only needed to happen once. Now it should use Mongo by default.

---

Good catch, in fact the values in this file are only used in integration tests, but good to have them aligned with what will be deployed to prod.